### DISCUSSION TITLE
Missing -AddAppRegistrationToWifConfig

### DISCUSSION THREAD

**Discussion by btarney** [d365collaborative/d365fo.tools#811] [General]
I am playing around with the new EntraIntegration function and am missing the flag for add the Wif Config. Confirmed that I have installed the latest 0/7.9 version. Is there something I am missing?

![Capture](https://github.com/d365collaborative/d365fo.tools/assets/161051335/c106ca3e-1165-4220-a48c-f354cdca4464)

**Comment by FH-Inway** [comment]
Thanks for reporting. You are a bit ahead of us, while the pull request #809 for adding the `-AddAppRegistrationToWifConfig` switch has been merged, a new version with this change has not been released to the PowerShell Gallery yet. Version 0.7.9 is the version before that change was made, which is why the switch is not available in that version.

@Splaxi Any chance you can make a release for the latest changes?

**Comment by FH-Inway** [comment]
@btarney In the mean time, you can use the script I mention in https://github.com/d365collaborative/d365fo.tools/issues/796#issuecomment-1872985731 to do what the switch does.

**Comment by FH-Inway** [comment]
`-AddAppRegistrationToWifConfig` is now available in version [0.7.10](https://github.com/d365collaborative/d365fo.tools/releases/tag/0.7.10) of d365fo.tools.
